cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -491,6 +491,7 @@ class CourseAboutSearchIndexer(object):
         AboutInfo("enrollment_end", AboutInfo.PROPERTY, AboutInfo.FROM_COURSE_PROPERTY),
         AboutInfo("org", AboutInfo.PROPERTY, AboutInfo.FROM_COURSE_PROPERTY),
         AboutInfo("modes", AboutInfo.PROPERTY, AboutInfo.FROM_COURSE_MODE),
+        AboutInfo("language", AboutInfo.PROPERTY, AboutInfo.FROM_ABOUT_INFO),
```

Comment:
I though that we'd decided upon adding it as a course property... in which case, it seems like it should be `AboutInfo.FROM_COURSE_PROPERTY` here... not from about information. Indeed, I was just asked to ensure that it becomes a course property, so we'll have to change it here anyway.